README.md

```diff
@@ -1,4 +1,11 @@
+## KanyonKris Fork ##
+### Changes ###
+- Added WiFiManager. A better way to setup the WiFi connection. Creates an Access Point portal the user connects to (Info-Orbs) and selects/enters the SSID and password of their WiFi Router. Then switches to station mode and connects to the router. If it can't connect to the router it switches to AP portal mode so the user can select/enter a different SSID and password. Much better than hardcoded credentials in a config.h file.
 
+### To Do ###
+- Add settings parameters to the portal so all/most of the user settings in the config.h can be changed in the portal.
+
```

Comment:
This change is probably not intended for the PR?

Reply:
Correct. I though I was just changing my forked copy of the README. I didn't know this would be included in the pull request. I'm a bit of a Git newbie. Sorry about that.

Reply:
Usually you want to create a branch per feature, so that if you create a PR you can do it from this branch.
If there is some feedback on your PR, you can push new changes to this branch and the PR will be updated.

Reply:
OK. So what now? Abandon this pull request? On my fork create a branch then do a new pull request?

Reply:
> OK. So what now? Abandon this pull request? On my fork create a branch then do a new pull request?

Up to you,
you can keep your `main` clean until this PR is resolved, or you can re-create the PR from a feature branch (I would recommend this last one)